src/mscorlib/src/Microsoft/Win32/UnsafeNativeMethods.cs

```diff
@@ -54,14 +49,6 @@ internal static class UnsafeNativeMethods
         [DllImport(Win32Native.USER32, EntryPoint = "LoadStringW", SetLastError = true, CharSet = CharSet.Unicode, ExactSpelling = true, CallingConvention = CallingConvention.StdCall)]
         internal static extern int LoadString(SafeLibraryHandle handle, int id, [Out] StringBuilder buffer, int bufferLength);
```

Comment:
`LoadString(SafeLibraryHandle` also needs to be factored out together with the rest of SafeLibraryHandle - to fix build break on Unix.